scripts/save_dataframe.py

```diff
@@ -0,0 +1,6 @@
+import config
```

Comment:
No big need to have a separate file for just saving function.

All saving/loading functions could live in one file